Description: Rename applications.menu file to prevent it clashing with
other desktops.  Longstanding issue with XDG menu spec which is
deliberately not changed so unable to forward upstream.
Forwarded: not-needed
Author: Jonathan Riddell <jr@jriddell.org>

Index: kservice-4.100.0/src/kbuildsycoca/CMakeLists.txt
===================================================================
--- kservice-4.100.0.orig/src/kbuildsycoca/CMakeLists.txt
+++ kservice-4.100.0/src/kbuildsycoca/CMakeLists.txt
@@ -32,5 +32,5 @@ install(TARGETS kbuildsycoca5 ${KF5_INST
 if (WIN32)
 install( FILES applications.menu  DESTINATION  ${SHARE_INSTALL_PREFIX}/xdg/menus )
 else ()
-install( FILES applications.menu  DESTINATION  ${SYSCONF_INSTALL_DIR}/xdg/menus )
+install( FILES applications.menu  DESTINATION  ${SYSCONF_INSTALL_DIR}/xdg/menus RENAME kf5-applications.menu )
 endif ()
Index: kservice-4.100.0/src/kbuildsycoca/kbuildsycoca.cpp
===================================================================
--- kservice-4.100.0.orig/src/kbuildsycoca/kbuildsycoca.cpp
+++ kservice-4.100.0/src/kbuildsycoca/kbuildsycoca.cpp
@@ -305,7 +305,7 @@ bool KBuildSycoca::build()
             g_vfolder->setTrackId(m_trackId);
         }
 
-        VFolderMenu::SubMenu *kdeMenu = g_vfolder->parseMenu("applications.menu");
+        VFolderMenu::SubMenu *kdeMenu = g_vfolder->parseMenu("kf5-applications.menu");
 
         KServiceGroup::Ptr entry = g_buildServiceGroupFactory->addNew("/", kdeMenu->directoryFile, KServiceGroup::Ptr(), false);
         entry->setLayoutInfo(kdeMenu->layoutList);
